x-pack/test/api_integration/apis/ml/annotations/update_annotations.ts

```diff
@@ -114,15 +84,16 @@ export default ({ getService }: FtrProviderContext) => {
 
       const originalAnnotation = annotationsForJob[0];
       const annotationUpdateRequestBody = {
-        timestamp: Date.now(),
-        end_timestamp: Date.now(),
-        annotation: 'Updated annotation #1',
+        ...commonAnnotationUpdateRequestBody,
         job_id: originalAnnotation._source.job_id,
         type: ANNOTATION_TYPE.ANNOTATION,
         event: 'model_change',
         detector_index: 2,
+        partition_field_name: 'airline',
+        partition_field_value: 'ANA',
```

Comment:
Since you've already added `...commonAnnotationUpdateRequestBody,` a few lines above I think you don't need to repeat the other fields.